Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift

```diff
@@ -66,7 +66,9 @@ class RubricCircleView: UIView {
         for i in 0..<count {
             let r = ratings[i]
             let description = descriptions[i]
-            let selected = i == (rubric?.selectedIndex ?? 0)
+
+            var selected = false
+            if let index = rubric?.selectedIndex, i == index { selected = true }
```

Comment:
this works, but `let selected = (i == rubric?.selectedIndex)` would be the same, right?